class.jetpack-gutenberg.php

```diff
@@ -89,6 +94,47 @@ class Jetpack_Gutenberg {
 	 */
 	private static $availability = array();
 
+	/**
+	 * Check to see if a block is able to run with current wp or gutenburg plugin versions
+	 *
+	 * @param array  $version_requirements An array containing wp and plugin version requirements.
+	 * @param string $extension_name The name of the block/plugin that has the version requirement.
+	 *
+	 * @since 8.4.0
```

Comment:
Maybe we can ship this in 8.3.0? It seems close to ready right now?

```suggestion
	 * @since 8.3.0
```

Reply:
Done